README.md

```diff
@@ -337,47 +338,6 @@ detector.with_api_token('foo').create()
 To add multiple alerting rules we would need to use different `detect`
 statements with distinct `label`s to differentiate them from one another.
 
-#### Detectors that Combine Data Streams
```

Comment:
Please pull in changes from master!

Reply:
I actually created a branch from master yesterday before working on this. Are you seeing any discrepancies?

Reply:
Well this entire section shouldn't be removed :P You can see it in [master here](https://github.com/Nike-Inc/signal_analog#detectors-that-combine-data-streams). Maybe your master was behind somehow?

Reply:
For some reason, I am not able to pull that from the master 😞  Added it manually